src/Domain/Creators/TimeDependence/RotationAboutZAxis.hpp

```diff
@@ -45,7 +45,7 @@ struct Inertial;
 
 namespace domain::creators::time_dependence {
 /*!
- * \brief A uniform rotation about the \f$z\f$ axis:
+ * \brief A rotation about the \f$z\f$ axis:
```

Comment:
How is the rotation no longer uniform? Even a global acceleration makes the rotation spatially uniform, which is how I always interpreted the "uniform" part. I think the shorter class name is fine either way since have a non-uniform rotation is kind of difficult/annoying.

Reply:
I always interpreted "uniform" for both rotation and translation to mean non-accelerating since `UniformTranslation` can only specify the velocity and before this change, `UniformRotation` could only specify angular velocity.

Reply:
Oh, translation is definitely meant as spatially uniform, temporal changes are sort of irrelevant for much of the code. I think only the control system cares, everything else only cares about the velocity.

Reply:
Yea, I wouldn't mind the docs being explicit like that :) But I think the class name is better. Is that okay with you?

Reply:
Yeah I'll just change the docs and leave the class name.